README.md

```diff
@@ -88,6 +88,27 @@ var_dump($token->validate($data)); // true, because current time is between "nbf
 $data->setCurrentTime(time() + 4000); // changing the validation time to future
 
 var_dump($token->validate($data)); // false, because token is expired since current time is greater than exp
+
+// We can also use leeway to deal with clock skew.
+// It uses the current time to validate, except that iat will add 5 seconds, nbf will add 10, and exp will subtract 20
+$dataWithLeeway = new ValidationData(null, ['iat' => 5, 'nbf' => 14, 'exp' => -20]); 
+$dataWithLeeway->setIssuer('http://example.com');
+$dataWithLeeway->setAudience('http://example.org');
+$dataWithLeeway->setId('4f1g23a12aa');
+
+var_dump($token->validate($dataWithLeeway)); // false, because token can't be used before now() + 60 and it's now() + 14
+
+$dataWithLeeway->setCurrentTime(time() + 51); // changing the validation time to future
```

Comment:
Use `$time = time()`, assigned only once to make it clear that we're working within the boundaries of fixed time throughout the example